routes/postsRoute.js

```diff
@@ -46,7 +46,7 @@ router.post(
 
 router.post(
   "/:id/comment",
```

Comment:
Is it right to create comments from posts-route? I think it would be more logical to do this from comments-route

Reply:
yes, that's exactly how it should be, otherwise we can write on a specific post, otherwise we have to connect the commentRoute and postRoute files, which is wrong